sdk/communication/Azure.Communication.Common/README.md

```diff
@@ -119,10 +119,31 @@ var entraTokenCredential = new InteractiveBrowserCredential(options);
 
 var entraTokenCredentialOptions = new EntraCommunicationTokenCredentialOptions(
     resourceEndpoint: "https://<your-resource>.communication.azure.com",
-    entraTokenCredential: entraTokenCredential
-){
-      Scopes = new[] { "https://communication.azure.com/clients/VoIP" }
-};
+    entraTokenCredential: entraTokenCredential,
+    scopes: new[] { "https://communication.azure.com/clients/VoIP" }
+);
+
+var credential = new CommunicationTokenCredential(entraTokenCredentialOptions);
+
+```
+
+The same approach can be used to authorize your Azure Communication Services resource to receive calls from the Teams resource account.
+This requires providing the `https://auth.msft.communication.azure.com/TeamsExtension.ManageCalls` scope
+```C# 
+var options = new InteractiveBrowserCredentialOptions
+    {
+        TenantId = "<your-tenant-id>",
+        ClientId = "<your-client-id>",
+        RedirectUri = new Uri("<your-redirect-uri>"),
+        AuthorityHost = new Uri("https://login.microsoftonline.com/<your-tenant-id>")
```

Comment:
Is the AuthorityHost required?

Reply:
That's actually a different library, Azure.Identity has login.microsoftonline.com as default and that should just work

https://learn.microsoft.com/en-us/dotnet/api/azure.identity.interactivebrowsercredentialoptions?view=azure-dotnet#properties

Reply:
(Azure.Identity wraps MSAL)